config/config.go

```diff
@@ -434,13 +438,18 @@ func LoadNode(path string, coordinator bool) (*Node, error) {
 			return nil, tracerr.Wrap(fmt.Errorf("error validating configuration file: %w", err))
 		}
 	}
+	log.Printf("Loaded Configuration: %+v", cfg)
```

Comment:
Will be great if this print "loaded configuration bla bla bla from config file|env|defaults"

Reply:
But this is the point. We MUST know where the parameter data came from, each one of them. Not only the error.